animations/cascaded-animation.html

```diff
@@ -79,11 +60,15 @@
         config.timing.delay += nodeDelay;
         config.node = node;
 
-        var animation = new animationConstructor();
-        var effect = animation.configure(config);
+        var animation = document.createElement(config.animation);
```

Comment:
This should only be performance negative iff a bad animation was configured. Latest commit addresses and and resets config state more completely as well.